### PR TITLE
ref(loader): Add analytics when loader is rendered

### PR DESCRIPTION
**Pull request by mydea**
This adds analytics for when the loader is rendered, so we can get better data in that area.

### CODE REVIEW

File: src/sentry/web/frontend/js_sdk_loader.py [hunk 1]
```diff
@@ -171,6 +188,17 @@ def get(self, request: Request, public_key: Optional[str], minified: Optional[st
...
 
+        analytics.record(
+            "js_sdk_loader.rendered",
+            org_id=key.project.organization_id,
+            is_lazy=loader_config["isLazy"],
+            has_performance=loader_config["hasPerformance"],
+            has_replay=loader_config["hasReplay"],
+            has_debug=loader_config["hasDebug"],
+            sdk_version=sdk_version,
+            tmpl_instance=instance,
```
Comment (AbhiPrasad) [inline src/sentry/web/frontend/js_sdk_loader.py:199]: m: Can we just store `tmpl` instead (`sentry/js-sdk-loader-noop.js.tmpl`)? Feel like that is easier to understand than default/noop/minified.

**Review by AbhiPrasad — COMMENTED** [review]
Perhaps @scefali can help us decide if this is ok amplitude cost wise.

File: src/sentry/web/frontend/js_sdk_loader.py (hunk 1 above)
Comment (mydea) [reply src/sentry/web/frontend/js_sdk_loader.py:199]: sure thing!

**Review by AbhiPrasad — APPROVED** [review]
LGTM! Might be interesting to watch the sentry dashboard (https://sentry.sentry.io/dashboard/38857/?project=1&statsPeriod=24h) and see if load times are affected by recording these metrics. I assume not, but maybe we discover something!

**Comment by mydea** [comment]
FYI after looking at the Big Query table & other example analytic events, we updated the naming to `organization_id` & also added `project_id` because these are handled specially.

**Review by AbhiPrasad — APPROVED** [review]

**Comment by priscilawebdev** [comment]
@mydea this is when the loader is rendered in the UI? why did we decide to go with analytics on the backend?

**Comment by mydea** [comment]
> @mydea this is when the loader is rendered in the UI? why did we decide to go with analytics on the backend?

Because it is the easiest way to capture this correctly, basically :)

**Pull request merged**